client/components/ProductInfo.jsx

```diff
@@ -1,18 +1,34 @@
-/* eslint-disable react/prop-types */
-/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Link from './Link';
```

Comment:
User defined components should always be uppercase and React related imports like React and PropTypes are uppercase by convention. Otherwise I'd say follow the documentation (for styled components and moment js for example always lowercase, and it might have something to do with what they export - component vs function..). Haven't found a good explanation online. I don't think it's super important tho.